docusaurus.config.js

```diff
@@ -2,224 +2,249 @@
-            items: [
-              {
-                label: 'Documentation',
-                to: '/docs/intro',
-              },
-              {
-                label: 'API Docs',
-                to: '/api',
-              },
-              {
-                label: 'SDK',
-                to: '/docs/architecture/sdk',
-              },
-              {
-                label: 'Engineering Principles',
-                href: 'https://github.com/epilot-dev/engineering-principles',
-              },
-            ],
-          },
-          {
-            title: 'Links',
-            items: [
-              {
-                label: '360 Portal',
-                href: 'https://portal.epilot.cloud',
-              },
-              {
-                label: 'GitHub',
-                href: 'https://github.com/epilot-dev',
-              },
-              {
-                label: 'Blog',
-                href: 'https://dev.to/epilot',
-              },
-              {
-                label: 'Tech Radar',
-                href: `${DOCS_URL}/techradar`,
-              },
-            ],
-          },
-        ],
-        copyright: `&copy; epilot.cloud ${new Date().getFullYear()}`,
-      },
-      prism: {
-        theme: lightCodeTheme,
-        darkTheme: darkCodeTheme,
-      },
-    }),
-});
+        footer: {
+          style: 'dark',
+          links: [
+            {
+              title: 'Docs',
+              items: [
+                {
+                  label: 'Documentation',
+                  to: '/docs/intro',
+                },
+                {
+                  label: 'API Docs',
+                  to: '/api',
+                },
+                {
+                  label: 'Graphql Docs',
+                  to: '/graphql/customer-portal',
+                },
+                {
+                  label: 'SDK',
+                  to: '/docs/architecture/sdk',
+                },
+                {
+                  label: 'Engineering Principles',
+                  href: 'https://github.com/epilot-dev/engineering-principles',
+                },
+              ],
+            },
+            {
+              title: 'Links',
+              items: [
+                {
+                  label: '360 Portal',
+                  href: 'https://portal.epilot.cloud',
+                },
+                {
+                  label: 'GitHub',
+                  href: 'https://github.com/epilot-dev',
+                },
+                {
+                  label: 'Blog',
+                  href: 'https://dev.to/epilot',
+                },
+                {
+                  label: 'Tech Radar',
+                  href: `${DOCS_URL}/techradar`,
+                },
+              ],
+            },
+          ],
+          copyright: `&copy; epilot.cloud ${new Date().getFullYear()}`,
+        },
+        prism: {
+          theme: lightCodeTheme,
+          darkTheme: darkCodeTheme,
+        },
+      }),
+  }
+);
```

Comment:
Is there any chance you could remove the formatting changes so we can see the actual changes? 🙏

Reply:
@anttiviljami Just disabled the prettier on the config file for now. Will format the file properly in a separate PR.